spec/pdfs/distribution_pdf_spec.rb

```diff
@@ -0,0 +1,39 @@
+# avoid infinite loop
```

Comment:
I'am not exactly sure what this is referring to

Reply:
I'll update the comment. Rubocop seems to have a problem with fixing this file.